spec/getCountriesSpec.js

```diff
@@ -135,4 +138,46 @@ describe("The index", () => {
       ).toBeTrue();
     });
   });
+
+  describe("The getAllCountriesByLandlock", () => {
+    it("returns all the landlocked countries", () => {
+      expect(countryApi.getCountriesByLandLock(true).length).toEqual(45);

```

Comment:
Can we move the `countryApi.getCountriesByLandLock(true)` (and the one with `false`) so it can be reused in other specs? (same for line: 144, 148, 152, 166)

Reply:
Agreed will do.

Reply:
Yeaa, that will be neat